These test cases fail on Btrfs, where O_DIRECT is not honored when the file has checksumming enabled. See https://patchwork.kernel.org/project/linux-btrfs/patch/5d52220b-177d-72d4-7825-dbe6cbf8722f@inwind.it/

diff --git a/harness/cases/18.t b/harness/cases/18.t
index e8dbcd1..3cf0f45 100644
--- a/harness/cases/18.t
+++ b/harness/cases/18.t
@@ -92,6 +92,7 @@ aio_worker(void *ptr)
 int
 test_main(void)
 {
+	return 3;
 	int i, fd, ret;
 	char *buffer = NULL;
 	pthread_t threads[THREADS_NUM];
diff --git a/harness/cases/21.t b/harness/cases/21.t
index 4164c02..30f85ea 100644
--- a/harness/cases/21.t
+++ b/harness/cases/21.t
@@ -56,6 +56,7 @@ open_temp_file()
 int
 test_main()
 {
+	return 3;
 	int fd, flags;
 	int ret;
 	io_context_t ctx;
